shutterstock-ruby.gemspec

```diff
@@ -4,7 +4,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'shutterstock-ruby'
-  s.version     = '0.2.0'
+  s.version     = '1.0.0'
```

Comment:
Is this change a breaking change?  Is it worth talking to upstream maintainer if so (or make it backward compatible, if it's not too hard)?

Reply:
Yeah everything was static before so it's a breaking change. I'll have a look to see if I can make it backwards compatible, but not sure if it's possible

Reply:
Might be able to keep the static stuff, and initialise a `Configuration` with those values if present?  Or just talk to the maintainer and see if s(he) thinks it's ok to have a breaking change.